spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/DetectionOutputSSD.scala

```diff
@@ -275,6 +275,14 @@ class DetectionOutputSSD[T: ClassTag](val nClasses: Int = 21,
     if (null != confPost) confPost.clearState()
     this
   }
+
+  override def computeOutputShape(inputShape: Shape): Shape = {
+    if (isTraining()) {
+      return inputShape
+    }
+    throw new RuntimeException("Haven't been " +
```

Comment:
Not implemented normally means it's expected to be implemented but not for now, i assume here not supported ?

Reply:
yeah, it means not supported here. Message should be changed.